File: SDKs/openaq.local/2.0.0/python/src/sdk/models/shared/openaqcitiesresult.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from . import meta
from . import citiesrow


@dataclass_json
@dataclass
class OpenAqCitiesResult:
    meta: Optional[meta.Meta] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'meta' }})
    results: List[citiesrow.CitiesRow] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'results' }})
    

File: SDKs/openaq.local/2.0.0/typescript/src/sdk/models/shared/openaqcitiesresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Meta } from "./meta";
import { CitiesRow } from "./citiesrow";



export class OpenAqCitiesResult extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=meta" })
  meta?: Meta;

  @SpeakeasyMetadata({ data: "json, name=results", elemType: CitiesRow })
  results: CitiesRow[];
}
